src/VirtoCommerce.Platform.Web/Controllers/Api/DiagnosticsController.cs

```diff
@@ -43,5 +46,20 @@ public ActionResult<License> GetSystemInfo()
             };
             return Ok(result);
         }
+
+        [HttpGet]
+        [Route("environmentInfo")]
+        [Authorize(PlatformConstants.Security.Permissions.EnvironmentInfoQuery)]
+        public ActionResult<EnvironmentInfo> GetEnvironmentInfo()
```

Comment:
ENV information must be part of the SystemInfo type and retrieved using the GetSystemInfo method.

Reply:
Done